torch/testing/_internal/common_methods_invocations.py

```diff
@@ -294,6 +294,8 @@ def method_tests():
         ('floor', (), NO_ARGS, 'scalar', (True,)),
         ('ceil', (S, S, S), NO_ARGS, '', (True,)),
         ('ceil', (), NO_ARGS, 'scalar', (True,)),
+        ('rad2deg', (S, S, S), NO_ARGS, '', (True,)),
```

Comment:
I'm not as familiar with this test as @albanD, but you might need to make this line 
```
('rad2deg', (S, S, S), NO_ARGS, '', (True, 'aten::mul')),
```

You can try that or we can wait for @albanD to make his recommendation during the week.

Reply:
I can't find autograd failure, only jit failure. Not sure what it is testing though :/

For the complex build issues, rebasing on top of master will fix them.

Reply:
Yeah, you're right. Not autograd, but jit. Do you know how I could replicate this test failure on my local build? As in what tests should I run to see this failure? @albanD

Reply:
I don't see the old failing builds any more. But from the `test/` folder, you should be able to run `python test_jit TestClass.test_name`.  The error in the build reports the test class and test name that failed.